lessons/pt/basics/sigils.md

```diff
@@ -9,8 +9,10 @@
 
 ## Overview sobre Sigils
 
-Elixir fornece uma sintaxe alternativa para representar e trabalhar com literais.
-Um sigil (símbolo especial) vai começar com um til `~` seguido por um caractere.
+Elixir fornece uma sintaxe alternativa para representar e trabalhar com literais, chamada de sigils.
+
+Um sigilo começa com um til `~` e é seguido por um identificador e um par de delimitadores. Antes do Elixir 1.15, o identificador deve ser um único caractere. A partir da versão 1.15, o identificador também pode ser uma sequência de vários caracteres maiúsculos.
```

Comment:
I guess we can maintain the usage of the word `sigil` to keep the same usage across the document.

```suggestion
Um sigil começa com um til `~` e é seguido por um identificador e um par de delimitadores. Antes do Elixir 1.15, o identificador deve ser um único caractere. A partir da versão 1.15, o identificador também pode ser uma sequência de vários caracteres maiúsculos.
```